README.md

```diff
@@ -35,6 +35,14 @@ Change to the project directory
   cd sertifikaattilukija
 ```
 
+### Environment variables
```

Comment:
Meillä on .env.example tiedosto minkä vois mainita tässä, unohdin näköjään aikasemmin lisätä installationiin kohdan

Copy .env.example to local .env file

```bash
     cp .env.example .env
```